src/component/pokemon/Pokemon.js

```diff
@@ -0,0 +1,77 @@
+import React, {useEffect, useState} from "react";
+import axios from "axios";
+import "./Pokemon.css";
+import {Link, useNavigate} from "react-router-dom";
+
+
+const Pokemon = ({name}) => {
+    const [ loading, setLoading ] = useState( false );
+    const [ error, setError ] = useState( false );
+    const [pokemon, setPokemon] = useState([]);
```

Comment:
Ik zou de default waarde aanpassen naar undefined, zodat je een duidelijker verschil hebt tussen of het al wél of niet is ingeladen
```suggestion
    const [pokemon, setPokemon] = useState();
```